modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/show_caps_lock.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const A_CODE = 'KeyA';
+const F11_CODE = 'F11';
+const NUMPAD_1_CODE = 'Numpad1';
+const SHIFT_CODE = 'ShiftLeft' || 'ShiftRight';
+const Z_CODE = 'KeyZ';
+
+export default function showCapsLock(event, spanId) {
+	const span = document.getElementById(spanId);
+
+	if (!span) {
+		throw new TypeError(
+			"the provided spanId doesn't return a valid element"
+		);
+	}
+
+	const {code} = event;
```

Comment:
![Screen Shot 2022-04-13 at 12 09 04 PM](https://user-images.githubusercontent.com/6843530/163130480-885c8297-e0f0-4736-a4da-7e6a92edc632.png)

We may not want to use "code" as it is specific to qwerty keyboards. I think the logic is as follows

- `key`: physical key represented by the event
- `code`: value of the key represented by the event

Reply:
This applies to all uses of code vs key in this file.

The old implementation, `keyCode` is deprecated in browsers

Reply:
Fair, I did consider both but Diego's comment swayed me towards `code`, thanks 👍